.bingo/kustomize.mod

```diff
@@ -2,4 +2,4 @@ module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
 go 1.18
 
-require sigs.k8s.io/kustomize/kustomize/v3 v3.8.7
+require sigs.k8s.io/kustomize/kustomize/v4 v4.5.5
```

Comment:
I thought that `kustomize`'s max version is 4.5.7?

Reply:
I just used 4.5.5 because it was specifically mentioned in the operator-sdk upgrade instructions. Updated to 4.5.7 now :+1: